### ISSUE TITLE
[CLI]: add lima

### ISSUE THREAD

**Issue by developer-guy** [alexellis/arkade#617]
## Expected Behaviour

Install [lima](https://github.com/lima-vm/lima) through arkade

## Current Behaviour

Not supported yet

## Are you a GitHub Sponsor (Yes/No?)

Check at https://github.com/sponsors/alexellis

- [ ] Yes
- [x] No

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* What Kubernetes distribution are you using?

```
kubectl version
```

* Operating System and version (e.g. Linux, Windows, MacOS):

```
uname -a

cat /etc/os-release
```

* What arkade version is this?

```
arkade version
```

**Comment by developer-guy** [comment]
hello @alexellis, IMO, I have a problem when I want to install lima via arkade because lima depends on another called limactl which is packaged along with lima tool. In arkade, we extract the .tar.gz file into tmp/ folder, then we move the only lima binary into /root/.arkade/bin. when we do that, because the lima binary depends on limactl binary, we can't run either lima or limactl. So, I've opened an issue on the lima side to tell the problem we faced with.
